images/logo-white.svg

```diff
@@ -1,10 +1 @@
-<?xml version="1.0" encoding="UTF-8"?>
```

Comment:
The logo images are likely (i.e: should be) plain copies of images from the style repo. I would encourage that they not be modified here. If you feel compelled to optimise them, I suggest looking at that repo instead.
The value of them being exact copies of the files from the style repo almost certainly outweighs the minor size benefit from this change (especially as they'll be served GZIPPed anyway).